docs/reference/migration/migrate_7_16.asciidoc

```diff
@@ -296,6 +298,29 @@ your cluster settings through the `PUT _cluster/settings` REST API. When modifyi
 use only persistent settings.
 ====
 
+[discrete]
+[[breaking_716_indices_deprecations]]
+==== Indices deprecations
+
+[[deprecation-system-indices]]
+.Direct access to system indices is deprecated.
+[%collapsible]
+====
+*Details* +
+Directly accessing system indices is deprecated. In {es} 8.0, you must add the
+`allow_restricted_indices` permission set to `true` on a user role to access
+system indices. Refer to 
+{ref}/defining-roles.html#roles-indices-priv[indices privileges] for
+information on adding this permission to an index privilege.
+
+*Impact* +
+Accessing system indices directly results in warnings in the header of API 
+responses. Use {kib} or the associated feature's {es} APIs to manage the data 
```

Comment:
Do we need to mention the deprecation logs, or is that implied?

Reply:
@williamrandolph, we could clarify this statement and say:

>Accessing system indices directly results in warnings in the header of API responses and in the deprecation logs.